js/ui/configuration-popup.js

```diff
@@ -75,14 +76,14 @@ Vue.component('configuration-popup', {
       let newValue = parseFloat(this.uiResolution) + increment
       newValue = this.withinRange(document.getElementById('resolution_input'), newValue)
       this.uiResolution = newValue
-      this.$forceUpdate()
+      // this.$forceUpdate()
```

Comment:
Is there a particular reason why many of those lines are left as comments? If not, just remove.